addons/l10n_in/models/res_partner.py

```diff
@@ -18,6 +18,15 @@ class ResPartner(models.Model):
             ('deemed_export', 'Deemed Export'),
         ], string="GST Treatment")
 
+    fiscal_country_code = fields.Char(compute='_compute_fiscal_country_code')
+
+    @api.depends('company_id')
+    @api.depends_context('allowed_company_ids')
+    def _compute_fiscal_country_code(self):
+        for record in self:
+            company = self.env.company
+            record.fiscal_country_code = company.account_fiscal_country_id.code
+
```

Comment:
You should move this to `partner.py` in the `account` module and rename the function `_compute_fiscal_country_codes` as it is what seems to be used. Also, use the exact same function as in master so when the forward port reaches master we should have no/minimal conflicts.

Reply:
@rugo-odoo I implemented it this way because 15.0 doesn't support (not ilike)

Reply:
By replacing the `l10n_in_gst_treatment` invisible value with what I added (now) on the suggestion, it should work with the same function here.

So you can apply all the changes requested.

Reply:
By replacing the `l10n_in_gst_treatment` invisible value with what I added (now) on the suggestion, it should work with the same function here.

So you can apply all the changes requested.